components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -1191,7 +1192,7 @@ subroutine Allocation2_ResolveNPLimit (bounds, num_soilc, filter_soilc  , &
                   ! calculate competition coefficients for NH4/NO3
                   ! first need to convert concentration to per soil water based
                   ! 2.76 consider soil adsorption effect on [NH4+] availability, based on Zhu et al., 2016 DOI: 10.1002/2016JG003554
-                  solution_nh4conc(c,j) = smin_nh4_vr(c,j) / (bd(c,j)*2.76 + h2osoi_vol(c,j)) ! convert to per soil water based
+                  solution_nh4conc(c,j) = smin_nh4_vr(c,j) / (bd(c,j)*2.76*1.0e-3_r8 + h2osoi_vol(c,j)) ! convert to per soil water based
```

Comment:
@qzhu-lbl @rgknox It appears this has already been fixed in master. Correct?

Reply:
yes this is in, but @qzhu-lbl  should have the final word on what this new scalar value should be... I think we are both using 1e-3 ....

Reply:
@bishtgautam @rgknox yes, 1e-3 is the correct scalar to use.